DuckDuckGo/Preferences/Model/VPNPreferencesModel.swift

```diff
@@ -251,7 +251,21 @@ final class VPNPreferencesModel: ObservableObject {
         return alert
     }
 
-    // MARK: - Excluded Sites
+    // MARK: - Actions
+
+    @MainActor
+    func manageExcludedApps() {
+        let windowController = ExcludedAppsViewController.create().wrappedInWindowController()
+
+        guard let window = windowController.window,
+              let parentWindowController = WindowControllersManager.shared.lastKeyMainWindowController
+        else {
+            assertionFailure("DataClearingPreferences: Failed to present ExcludedDomainsViewController")
```

Comment:
Just realized the message is slightly off here, so I'm marking this to remember to fix.

Reply:
Fixed.